operations/system_admin_toolkit/about_sat/Introduction_to_SAT.md

```diff
@@ -1,135 +1,135 @@
 # Introduction to SAT
 
-## About System Admin Toolkit (SAT)
-
 The System Admin Toolkit (SAT) is designed to assist administrators with common tasks, such as
 troubleshooting and querying information about the HPE Cray EX System and its components, system
 boot and shutdown, and replacing hardware components.
 
-### SAT Command Line Utility
+## SAT Command-Line Utility
 
-The System Admin Toolkit (SAT) provides a command-line utility called `sat` that can be run from
-Kubernetes control plane nodes (`ncn-m` nodes). The `sat` command-line utility is organized into
-multiple subcommands that perform different administrative tasks. For example, `sat status` provides
-a summary of the status of the components in the system while `sat bootprep` provides a way to
+The System Admin Toolkit (SAT) provides a command-line utility called `sat` that runs from
+Kubernetes control plane nodes (NCNs). The `sat` command-line utility is organized into
```

Comment:
This edit loses some meaning. NCN literally means non-compute nodes and is often used to refer to the management nodes specifically. If we take that narrower definition, it would include the master/control-plane nodes (i.e. nodes that have host names starting with `ncn-m`), the Ceph storage nodes (nodes with host names starting with `ncn-s`), and worker nodes (nodes with host names starting with `ncn-w`).

The `sat` command is installed on control-plane/master nodes (`ncn-m*`) and technically also on worker nodes (`ncn-w*`), but we have not really advertised it on worker nodes.

Reply:
Reverting back to "(`ncn-m` nodes)" in the next commit. I think that fixes it.